com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/ZigBeeNetworkManager.java

```diff
@@ -264,6 +264,7 @@ public ZigBeeNetworkManager(final ZigBeeTransportTransmit transport) {
         transitions.put(ZigBeeNetworkState.INITIALISING, new HashSet<>(Arrays.asList(ZigBeeTransportState.ONLINE)));
         transitions.put(ZigBeeNetworkState.ONLINE, new HashSet<>(Arrays.asList(ZigBeeTransportState.OFFLINE)));
         transitions.put(ZigBeeNetworkState.OFFLINE, new HashSet<>(Arrays.asList(ZigBeeTransportState.ONLINE)));
+        transitions.put(ZigBeeNetworkState.SHUTDOWN, new HashSet<>(Arrays.asList(ZigBeeTransportState.OFFLINE)));
```

Comment:
I'm not sure that this is correct - this will allow the system to be restarted, and that is not allowed. How about adding an empty set here instead?

Reply:
This means there is no allowed transition from SHUTDOWN to any state? In this case I would suggest to not add any transition to the collection and instead change the method `setTransportState(final ZigBeeTransportState state)` to do a null check. WDYT?

Reply:
> This means there is no allowed transition from SHUTDOWN to any state?

That's correct - as per the state transition diagram -:

https://github.com/zsmartsystems/com.zsmartsystems.zigbee/blob/master/com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/doc-files/ZigBeeNetworkStateTransitions.png

![image](https://user-images.githubusercontent.com/1041475/58541431-69963b00-81f3-11e9-935c-93da7e124929.png)


> In this case I would suggest to not add any transition to the collection and instead change the method setTransportState(final ZigBeeTransportState state) to do a null check. WDYT?

I don't mind too much. Just adding the empty Set seems ok to me - it's (arguably) cleaner, and does make it clear if someone looks at all the allowed state transitions, that SHUTDOWN has no allowable states.

If you think the null check is simpler though, I don't mind, but I think the code will look more cluttered ;)

Reply:
OK. Then I'll just add an empty set.

Reply:
I updated the PR